cms/djangoapps/contentstore/views/block.py

```diff
@@ -84,9 +86,10 @@
 # ends, which end up reading from an outdated state of the database. For more information see the discussion in the
 # following PR: https://github.com/openedx/edx-platform/pull/34020
 @transaction.non_atomic_requests
-@require_http_methods(("DELETE", "GET", "PUT", "POST", "PATCH"))
-@login_required
+# @require_http_methods(("DELETE", "GET", "PUT", "POST", "PATCH"))
+@view_auth_classes()
 @expect_json
```

Comment:
I feel like we can drop the `expect_json` decorator here, as django_rest_framework should handle json gracefully. Not sure if we can/should force json-only - I think django_rest_framework will also accept and parse form encoded data too.